src/SurfaceFluxes.jl

```diff
@@ -288,7 +288,7 @@ function surface_conditions(
     param_set::APS,
     sc::AbstractSurfaceConditions{FT},
     scheme::SolverScheme = FVScheme();
-    tol::RS.AbstractTolerance = RS.RelativeOrAbsoluteSolutionTolerance(FT(3e-5), FT(Δz(sc) / 50)),
+    tol::RS.AbstractTolerance = RS.RelativeOrAbsoluteSolutionTolerance(100eps(FT), 10eps(FT)),
```

Comment:
These should not be FP dependent. A tolerance of something like 1e-2 on the Richardson number should suffice, independent of FP.